README.md

```diff
@@ -0,0 +1,44 @@
+# Softwire Morale Event Discord Bot
+
+This Discord bot makes it easier for morale event admins to organise events, and for people to sign up to those events.
+It links up with Google Sheets to generate an at-a-glance list of all the events happening, and has other features to make this all easier.
+
+## Setup
+
+### Creating a bot for local testing
+
+1. Have someone on the team add you to the SoftwireCambridge Discord developer team
+2. Go to https://discord.com/developers/applications and click "New Application" in the top right
+3. Name your local testing bot, making sure that the team is set to SoftwireCambridge
+![Create application](readme-pics/create-application.png)
+
+### Getting Google Sheets credentials
+
+1. Talk to someone on the team (currently Benji) who can give you Google sheets credentials, and ask for them.
+They will be given to you on [Zoho](https://vault.zoho.com/online/main), so make sure someone on the team raises a helpdesk ticket
+to add you to the "Morale Event Discord Bot" chamber
```

Comment:
I think you can request access yourself using HYP. So probably enough to just say to raise a HYP request to get access to the zoho chamber 🙂